recipes/r-maditr/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '0.8.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-maditr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/maditr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/maditr/maditr_{{ version }}.tar.gz
+  sha256: f06835f6d4b19cf163d6294ec20b792e6cb320699935635c99cc5ff2df2dafda
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-data.table >=1.12.6
+    - r-magrittr >=1.5
+  run:
+    - r-base
+    - r-data.table >=1.12.6
+    - r-magrittr >=1.5
+
+test:
+  commands:
+    - $R -e "library('maditr')"           # [not win]
+    - "\"%R%\" -e \"library('maditr')\""  # [win]
+
+about:
+  home: https://github.com/gdemin/maditr
+  license: GPL-2.0-or-later
```

cran meta data `# License: GPL-2` converts to
```suggestion
  license: GPL-2.0-only
```